examples/json-record-map-conversion/json_record_map_conversion.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE shows how to convert Ballerina records, maps, and JSON objects from one type to another.
+keywords: ballerina, ballerina by example, json, record, map
```

Comment:
Should we use "json object" instead
```suggestion
keywords: ballerina, ballerina by example, json object, record, map
```

Reply:
For this context, it's  fine to just say json right?

Reply:
which should we add here?

Reply:
@MaryamZi

Reply:
My original suggestion was because we can only convert a JSON object to a record or a map. Would we lose anything by making this "json object"? Even just "json" is fine.

Reply:
Not exactly sure. I was just wondering if our keyword is "json object", whether it'd be a match for just "json". IMO, no need to go into nuances in terms for this purpose